### ISSUE TITLE
[micromamba] install -f env.yml doesn't install into current prefix

### ISSUE THREAD

**Issue by wolfv** [mamba-org/mamba#1050]
when installing from an env.yml file, micromamba doesn't install into the activated prefix, but rather into the one indicated in the env.yml file.

I think `install` should always refer to the activated prefix unless explicitly stated.

What do you think @adriendelsalle?

**Comment by adriendelsalle** [comment]
I agree that `name` key of yaml spec files is ambiguous when used in `install` or `update` operations.

Do you mean we should not parse this key?
How to handle the case where the command is ran outside an already activated env? Use the `name` key only in that case?

**Comment by soerendip** [comment]
I ran into this often. The path in the file often does not make sense, because it it might ave been created by a different user. Then it is something like `/home/sven/miniconda/envs...`. But the current system actually has no user `sven`. 
So, taking the `name` actually makes more sense than the prefix. Then providing a name with `mamba create -n newenv -f env.yml` should install the packages from the file and ignore the prefix and name and instead use the default prefix and the provided name. 

However, what if the user only wants to add or exchange the libraries in the file in a currently active environment? `mamba install -f filename` I think should install in the currently active environment. That is what I would expect intuitively. `create` for creation of a new environment and `install` for adding libraries to an existing one. Either default or the one specified with `-n`.

 Though `mamba update...` is also fine. Best if all of this would be consistent with conda.
 Cheers

**Comment by sblask** [comment]
I would like the following to work:

```
$ micromamba create --prefix ./.conda_env
$ micromamba activate ./.conda_env
$ micromamba install --file environment.yml
```

even if there is a name specified in environment.yml. But if there is I get an error:

```
...
error    libmamba No prefix found at: /Users/sebastian/micromamba/envs/NAME_FROM_environment.yml
error    libmamba Environment must first be created with "micromamba create -n {env_name} ..."
critical libmamba Aborting.
info     libmamba ****************** Backtrace End ********************
```

I have to do this to get it to work:

```
`$ micromamba install --file environment.yml --prefix ./.conda_env`
```

**Comment by JohanMabille** [comment]
This has been fixed in mamba 2.
